index.js

```diff
@@ -590,13 +597,23 @@ function launchApp(argo, startExternalAdapterServer) {
 
         // this ensures that external connections that start the runtime can do so without a main window
         let successfulInitialLaunch = true;
-
+        let passedMutexCheck = false;
+        let failedMutexCheck = false;
+        if (uuid && !isRunning) {
+            if (!isUuidAvailable(uuid)) {
+                deleteApp(uuid);
```

Comment:
Which case is this, the uuid is in the local core state but not running but exists somewhere else? Aren't we cleaning them out when they close now?

Reply:
This is in case we didn't clear it out. We only provide a new api call to clear out, we don't do it on close yet.

Reply:
gotcha 👍